examples/Creating_slides_with_Assistants_API_and_DALL-E3.ipynb

```diff
@@ -0,0 +1,786 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's take a look at our image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Image](data/dalle_img.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Cool, now we can add this image to our thread, and ask the Assistant API to write the code to create our slide!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we can save the image locally, then upload it using the `File` upload endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dalle_img_path = 'data/dalle_image.png'\n",
+    "img = requests.get(image_url)\n",
+    "\n",
+    "#Save locally\n",
+    "with open(dalle_img_path,'wb') as file:\n",
+    "  file.write(img.content)\n",
+    "\n",
+    "#Upload\n",
+    "file = client.files.create(\n",
+    "  file=open(dalle_img_path, \"rb\"),\n",
+    "  purpose='assistants'\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can simply ask the Assistant API to create a PowerPoint using the content it has helped us generate so far. We generated the title DALL-E3 image separately, so we will explicitly upload the file, but otherwise will ask the Assistant to use the relevant data to generate two slides in whatever format we want."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Run(id='run_kPvYu6oKwMVmwocMRDqTm5IA', assistant_id='asst_7g0ZZvr42G9vxVBHyNCy2rIn', cancelled_at=None, completed_at=None, created_at=1701663863, expires_at=1701664463, failed_at=None, file_ids=['file-1JMqJZGth54cPsMKdwuCgmYG'], instructions='You are a data scientist assistant. When given data and a query, write the proper code and create the proper visualization', last_error=None, metadata={}, model='gpt-4-1106-preview', object='thread.run', required_action=None, started_at=None, status='queued', thread_id='thread_jpiHVuCY2Vr0wPXeLVk9zExB', tools=[ToolAssistantToolsCode(type='code_interpreter')])"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "submit_message(assistant.id,thread,\"Output a pptx slide deck with two slides as follows:\\\n",
+    "               MAKE SURE the first slide is titled 'NotRealCorp: Quarterly financial planning meeting, Q3 2023' at the top of the screen, with large heading font size. The image included should be large but positioned BELOW the text. IMPORTANT: Do not have the title text and images overlap. ALL SLIDES should have the top third JUST for Titles. Make sure the image is not taking up the whole screen, so the reader can see the entire title. The second\\\n",
```

Comment:
1/ have you tried specifying a font size? 
2/ could we potentially extract this into a template that can be re-used somehow?
3/ does it tend to regress without capitalizing "MAKE SURE" and "CENTERED"?